app/server/lib/ActiveDoc.ts

```diff
@@ -666,6 +667,7 @@ export class ActiveDoc extends EventEmitter {
             return this._afterMigration(docSession, 'storage',  newVersion, success);
           },
         });
+        this._registerSQLiteDB();
```

Comment:
Does this need to run in the if branch too?

Reply:
Oh, thank you! I moved the call just outside.